.gitmodules

```diff
@@ -4,7 +4,7 @@
 [submodule "lib/arbitrum-sdk"]
 	path = lib/arbitrum-sdk
 	url = https://github.com/OffchainLabs/arbitrum-sdk
-[submodule "lib/nitro-contracts.git"]
+[submodule "lib/nitro-contracts"]
 	path = lib/nitro-contracts
-  url = https://github.com/EspressoSystems/nitro-contracts
-  branch = celestia-integration
+	url = https://github.com/EspressoSystems/nitro-contracts
```

Comment:
why are we changing this?

Reply:
This was a result of fixing the nitro-contracts dependency via forge remove and forge install. I chose to pin this to the celestia integration branch as I was under the impression that is the one we are using as a "release" branch for our current work.

Reply:
These submodules are handled by forges dependency system. This was the result of removing the previous dependency that had a naming issue, and re-running forge install targeting the celestia-integration branch of our contracts. This is to keep everything in this branch, and then the celestia-integration branch of orbit-actions in sync while we develop. As we merge this into a release branch, we should add a commit to re-target this.